Polyhedron/demo/Polyhedron/CMakeLists.txt

```diff
@@ -387,122 +300,41 @@ if(CGAL_Qt5_FOUND AND Qt5_FOUND AND OPENGL_FOUND AND QGLVIEWER_FOUND)
-
-  qt5_wrap_ui(point_set_upsamplingUI_FILES Polyhedron_demo_point_set_upsampling_plugin.ui)
-  polyhedron_demo_plugin(point_set_upsampling_plugin Polyhedron_demo_point_set_upsampling_plugin ${point_set_upsamplingUI_FILES})
-  target_link_libraries(point_set_upsampling_plugin scene_points_with_normal_item)
-
-  qt5_wrap_ui( ps_outliers_removal_UI_FILES Polyhedron_demo_point_set_outliers_removal_plugin.ui)
-
-  polyhedron_demo_plugin(point_set_outliers_removal_plugin Polyhedron_demo_point_set_outliers_removal_plugin ${ps_outliers_removal_UI_FILES}) 
-  target_link_libraries(point_set_outliers_removal_plugin scene_points_with_normal_item)
-  
+  add_subdirectory(Point_set_simplification_plugin)
+  add_subdirectory(Point_set_selection_plugin)
+  add_subdirectory(Point_set_shape_detection_plugin)
+  add_subdirectory(Point_set_wlop_plugin)
+  add_subdirectory(Point_set_bilateral_smoothing_plugin)
+  add_subdirectory(Point_set_upsampling_plugin)
+  add_subdirectory(Point_set_outliers_removal_plugin)
   if(EIGEN3_FOUND AND ${EIGEN3_VERSION} VERSION_GREATER "3.1.90")
-    polyhedron_demo_plugin(hole_filling_plugin Polyhedron_demo_hole_filling_plugin ${hole_fillingUI_FILES})
-    target_link_libraries(hole_filling_plugin scene_polyhedron_item scene_polylines_item)
-
-    polyhedron_demo_plugin(hole_filling_polyline_plugin Polyhedron_demo_hole_filling_polyline_plugin)
-    target_link_libraries(hole_filling_polyline_plugin scene_polyhedron_item scene_polylines_item)
-
-    polyhedron_demo_plugin(fairing_plugin Polyhedron_demo_fairing_plugin ${fairingUI_FILES})
-    target_link_libraries(fairing_plugin scene_polyhedron_selection_item)
+    add_subdirectory(Hole_filling_plugin)
+    add_subdirectory(Fairing_plugin)
   else()
     message(STATUS "NOTICE: The hole filling and fairing plugins require Eigen 3.2 (or higher) and will not be available.")
   endif()
-
-  polyhedron_demo_plugin(selection_plugin Polyhedron_demo_selection_plugin ${selectionUI_FILES})
-  target_link_libraries(selection_plugin scene_polyhedron_selection_item scene_points_with_normal_item scene_polylines_item)
-  
-  polyhedron_demo_plugin(shortest_path_plugin Polyhedron_demo_shortest_path_plugin ${shortestPathUI_FILES})
-  target_link_libraries(shortest_path_plugin scene_polyhedron_item scene_polylines_item scene_polyhedron_selection_item scene_polyhedron_shortest_path_item scene_basic_objects)
-
-  qt5_wrap_ui( exampleUI_FILES Polyhedron_demo_example_plugin.ui dock_example.ui)
-  qt5_wrap_ui( example_dockUI_FILES dock_example.ui)
-  polyhedron_demo_plugin(example_plugin Polyhedron_demo_example_plugin ${exampleUI_FILES} ${example_dockUI_FILES})
-  target_link_libraries(example_plugin scene_polyhedron_item)
+  add_subdirectory(Selection_plugin)
+  add_subdirectory(Shortest_path_plugin)
+  add_executable(example_plugin IMPORTED)
+  set_property(TARGET example_plugin PROPERTY IMPORTED_LOCATION "${CMAKE_CURRENT_SOURCE_DIR}/../../../Example_plugin/example_plugin.cmake")
```

Comment:
I think that those two lines about `example_plugin` are:
  - not working:
    - the `example_plugin` is not an executable,
    - its `IMPORTED_LOCATION` cannot be a CMake file.
  - useless: the polyhedron demo does not need the list of plugins at compile time, they are discovered at runtime.